source/components/handling-events.md

```diff
@@ -0,0 +1,130 @@
+Instead of having to register event listeners on elements you'd like to
+respond to, simply implement the name of the event you want to respond to
+as a method on your component.
+
+For example, imagine we have a template like this:
+
+```hbs
+{{#x-clickable}}
+  This is a clickable area!
+{{/x-clickable}}
+```
+
+Let's implement `x-clickable` such that when it is
+clicked, an alert is displayed:
+
+```app/components/x-clickable.js
+export default Ember.Component.extend({
+  click: function(evt) {
+    alert("XClickableComponent was clicked!");
+  }
+});
+```
+
+Browser events may bubble up from the target component to each parent component
+in succession, until the root component. To enable bubbling `return true;` from
+the event handler method in your component.
+
+```app/components/x-clickable.js
+export default Ember.Component.extend({
+  click: function(evt) {
+    Ember.Logger.info("XClickableComponent was clicked!");
+    return true;
+  }
+});
+```
+
+### Event Names
+
+  All of the event handling approaches described above respond to the same set
+  of events. The names of the built-in events are listed below. (The hash of
```

Comment:
I don't like the leak of implementation detail here (both `Ember` classes), I would remove it.
It might be nice to have a section of registering custom events, pending team consensus.